plugins/tomcat.yaml

```diff
@@ -63,6 +63,15 @@ pipeline:
     timestamp:
       parse_from: timestamp
       layout: '%d/%b/%Y:%H:%M:%S %z'
+    severity:
+      parse_from: http_status
+      preserve_to: http_status
+      mapping:
+        error: 
+          - 4xx
+          - 5xx
+        info: 3xx
+        debug: 2xx
```

Comment:
In the `apache_http` plugin we used the following values should we be consistent in this behavior?
```
      mapping:
        info: 2xx
        notice: 3xx
        warning: 4xx
        error: 5xx
```

Reply:
Good point. I'll change that